cdap-api/src/main/java/io/cdap/cdap/api/app/ApplicationSpecification.java

```diff
@@ -63,6 +63,12 @@ public interface ApplicationSpecification {
   @Nullable
   String getConfiguration();
 
+  /**
+   * @return Change summary entered by the user during the update of the Application.
+   */
+  @Nullable
+  String getChangeSummary();
```

Comment:
add a default implementation `return null` here to preserve backward compatibility since this is in api class